apps/web/src/illustrations/text.tsx

```diff
@@ -0,0 +1,3 @@
+const IllustrationText: React.FC = () => <div />;
+
+export default IllustrationText;
```

Comment:
was this component supposed to have some content? why an empty `div`?

Reply:
hm there seems to be many of them, any purpose?

Reply:
I think Lucas added it just as a placeholder for the illustrations he's going to make stil